tests/unit/charm/test_status_vs_config.py

```diff
@@ -50,12 +51,8 @@ def test_unit_is_blocked_if_no_config_provided(self, is_leader, num_units):
 
             # AND the current unit could be either a leader or not
             self.harness.set_leader(is_leader)
-            self.harness.begin_with_initial_hooks()
-
-            # AND storage is attached
             self.harness.add_storage("content-from-git", attach=True)
-
-            self.harness.container_pebble_ready("git-sync")
+            self.harness.begin_with_initial_hooks()
```

Comment:
FYI @lucabello, most unit tests passed as soon as:
1. `container_pebble_ready` was removed; and
2. `add_storage` moved above `begin_with_initial_hooks`.